lib/rex/post/sql/ui/console/command_dispatcher/local_fs.rb

```diff
@@ -0,0 +1,41 @@
+# -*- coding: binary -*-
+
+require 'rex/post/local_fs'
+
+module Rex
+  module Post
+    module Sql
+      module Ui
+        module Console
+          module CommandDispatcher
+            #
+            # SQL File system commands
+            #
+            module LocalFs
```

Comment:
We'll need to swap this to `class` to be reused elsewhere when we put it in the more global generic location